src/app/modules/auth/AuthController.js

```diff
@@ -0,0 +1,18 @@
+const bcrypt = require('bcrypt')
+const jwt = require('jsonwebtoken')
+
+const RegisterDTO = require('./dtos/RegisterDTO')
+const serviceLocator = require('../../services/serviceLocator')
+
+class AuthController {
+  static async register(request, reply) {
+    const registerDto = RegisterDTO.fromRequest(request)
+
+    const authService = serviceLocator.authService()
+    const token = await authService.register(registerDto)
```

Comment:
Handles the '/auth/register' route. Maps a request onto a data transfer object and passes that to the auth service.